internal/querynodev2/services.go

```diff
@@ -1011,10 +999,18 @@ func (node *QueryNode) SyncDistribution(ctx context.Context, req *querypb.SyncDi
 
 	for nodeID, infos := range addSegments {
 		err := shardDelegator.LoadSegments(ctx, &querypb.LoadSegmentsRequest{
-			Base:         req.GetBase(),
-			DstNodeID:    nodeID,
+			Base: commonpbutil.NewMsgBase(
+				commonpbutil.WithMsgType(commonpb.MsgType_LoadSegments),
+				commonpbutil.WithMsgID(req.Base.GetMsgID()),
+			),
 			Infos:        infos,
+			Schema:       req.GetSchema(),
+			LoadMeta:     req.GetLoadMeta(),
 			CollectionID: req.GetCollectionID(),
+			ReplicaID:    req.GetReplicaID(),
+			DstNodeID:    nodeID,
+			Version:      time.Now().UnixNano(),
+			NeedTransfer: false,
```

Comment:
`Version` shall be `qc` allocated value.

Reply:
yes, that's truly a defect, I will change this